### PR TITLE
added VideoPage

### PR DESCRIPTION
**Pull request by shoqqan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function App() {
             <div className={styles.parent}>
                 <GlobalStyles/>
                 <Header changeTheme={changeTheme}/>
-                <Main/>
+                <VideoPage/>
```
Comment (hj-abzal) [inline src/App.js:34]: videopage is not reusable

File: src/components/VideoPage/VideoPage.module.scss [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
     h1 {
       color: $bgPrimaryColor;
+      color: #FEFEFE;
```
Comment (hj-abzal) [inline src/components/VideoPage/VideoPage.module.scss]: lmao bro use variable..

File: src/App.js (hunk 1 above)
Comment (shoqqan) [reply src/App.js:34]: bro u really suck, we can not use props

File: src/components/VideoPage/VideoPage.module.scss (hunk 2 above)
Comment (shoqqan) [reply src/components/VideoPage/VideoPage.module.scss]: fixed

File: src/App.js (hunk 1 above)
Comment (shoqqan) [reply src/App.js:34]: fixed, sometimes u suck, sometimes u monke

**Review by hj-abzal — APPROVED** [review]

**Pull request merged**